server/lib/host-metrics.js

```diff
@@ -266,6 +267,56 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(
+  host,
+  { startDate, endDate, collectiveIds = null, timeUnit } = {},
+) {
+  if (!collectiveIds) {
+    const hostedCollectives = await host.getHostedCollectives();
+    collectiveIds = hostedCollectives.map(c => c.id);
```

Comment:
I'm not sure about this part. We're going to load the full list of Collectives (in the case of OSC, 4000+) in RAM with their `longDescription`, `settings` and all the heavy stuff then populate that into the model just to get an `id`. I don't want to block on this because this PR is old and we really need to move it forward, but we should:
1. At least add an `attributes: ['id']` to avoid fetching unnecessary fields 
2. Have a dedicated query that returns just an array of integers
3. Do we really need that? Isn't filtering on `HostCollectiveId` enough?

Reply:
@Betree : Ah, good point. I think we can remove this one and instead use a template string to remove the collective id filtering from the query when `collecticeIds` are empty. I've made the change. Let me know if that looks good. 😃

Reply:
@znarf wdyt about that? Is it ok to not filter on `CollectiveId`?

Reply:
If we want to be consistent in our numbers, we currently have to pass the `collectiveIds` to match what we're doing in the totalMoneyManaged metric (`lib/budget` `getTotalMoneyManagedAmount`). In simple cases, it's the same number, but it's better to be on the safe side and have the exact same behavior.

Reply:
Alright, then @SudharakaP let's go with one of the other proposals. Just make sure we only load the IDs and not the full models

Reply:
@Betree : I've just added a dedicated query. Let me know how it looks now. Thanks. 😃